shared/src/test/scala/mlscript/DiffTests.scala

```diff
@@ -77,8 +77,10 @@ class DiffTests extends org.scalatest.funsuite.AnyFunSuite {
       expectTypeErrors: Bool, expectWarnings: Bool, expectParseErrors: Bool,
       fixme: Bool, showParse: Bool, verbose: Bool, noSimplification: Bool,
       explainErrors: Bool, dbg: Bool, fullExceptionStack: Bool, stats: Bool,
-      noExecution: Bool, noGeneration: Bool, showGeneratedJS: Bool)
-    val defaultMode = Mode(false, false, false, false, false, false, false, false, false, false, false, false, false, false)
+      noExecution: Bool, noGeneration: Bool, showGeneratedJS: Bool,
+      expectRuntimeError: Bool)
```

Comment:
This seems unused.

Reply:
Ah, it's used in the following commits.

Reply:
It’s used in runtime error flags. I’m marking this as resolved.